### PR TITLE
Sailfish SDK: Update README to point to sdk-build-tools

### PR DESCRIPTION
**Pull request by vigejolla**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -224,13 +224,23 @@ or using shadow builds.
 
 ## Building for Sailfish SDK
 
-1. create set of directories like '~/QtCreator/Build' '~/QtCreator/Install'
-2. navigate to '~/QtCreator' and clone the Qt Creator's source tree there
-3. run qmake:
-	qmake ../digia-qt-creator/qtcreator.pro -r -after "DEFINES+=IDE_COPY_SETTINGS_FROM_VARIANT=. IDE_SETTINGSVARIANT=SailfishBeta6" QTC_PREFIX=
+Building Qt Creator for the Sailfish SDK is done using some helper scripts,
```
Comment (martyone) [inline README.md:227]:
```suggestion
Building Qt Creator for the Sailfish SDK is done using tools,
```

The term scripts have some negative connotations :)

File: README.md [hunk 2]
```diff
@@ -224,13 +224,23 @@ or using shadow builds.
...
+which are available in a github repo:
 
-4. build QtCreator, intaller, artifacts etc as defined in
-	http://qt-project.org/wiki/Building-Qt-Creator-Packages
+    https://github.com/sailfishos/sdk-build-tools.git
+
+Before building Qt Cretor, you must build the prerequisite libraries (ICU, Qt
```
Comment (martyone) [inline README.md:232]:
```suggestion
Before building Qt Creator, you must build the prerequisite libraries (ICU, Qt
```

File: README.md [hunk 3]
```diff
@@ -224,13 +224,23 @@ or using shadow builds.
...
+and LLVM/Clang). In the abovementioned repository you can find scripts for
```
Comment (martyone) [inline README.md:233]:
```suggestion
and LLVM/Clang). In the abovementioned repository you can find tools for
```

File: README.md [hunk 4]
```diff
@@ -224,13 +224,23 @@ or using shadow builds.
...
+building each of them: buildicu.sh, buildqt5.sh and buildllvm.sh. Each of the
+shell scripts provide help when executed with the '--help' argument.
```
Comment (martyone) [inline README.md:235]:
```suggestion
build tools provide help when executed with the '--help' argument.
```

File: README.md (hunk 2 above)
Comment (vigejolla) [reply README.md:232]: This is weird, I noticed that typo when reading through what I had written, and I clearly remember fixing it :-O

File: README.md (hunk 1 above)
Comment (vigejolla) [reply README.md]: changed to tools

**Pull request merged**